distributed/config.py

```diff
@@ -97,6 +97,7 @@ def _initialize_logging_old_style(config: dict[Any, Any]) -> None:
     loggers: dict[str, str | int] = {  # default values
         "distributed": "info",
         "distributed.client": "warning",
+        "distributed.gc": "warning",
```

Comment:
This PR renames `utils_perf` to `gc`, both the file and the logger. This is breaking, but I don't think too many people use this. If anything, I can perceive people importing `disable_gc_diagnosis`. If worthwhile, we can run a deprecation cycle for these imports.

Reply:
From what I can tell, https://github.com/AlexeyPechnikov/pygmtsar/blob/b5c58d61f00350cc4115d63985af9caa2c7cd9de/pygmtsar/pygmtsar/IO.py#L455 is the only package with many users that does this.

Reply:
Is there any real benefit of renaming? Otherwise I'd keep as is if there are actual users out there. Not sure if this package counts as actual users?

Reply:
> Is there any real benefit of renaming? Otherwise I'd keep as is if there are actual users out there. Not sure if this package counts as actual users?

Mostly, I _really_ hate the name as it's completely non-descriptive.

Reply:
Once again, we could add a deprecation cycle, but I'd like to get rid of the name sooner or later.

Reply:
yeah lets change it now

Reply:
Can you open an issue there just to be nice?